src/tests/ftest/container/global_handle.py

```diff
@@ -32,8 +32,7 @@
 from apricot import TestWithServers
 
 import check_for_pool
-from daos_api import DaosContext, DaosPool, DaosContainer, DaosApiError
-from daos_cref import IOV
+from pydaos.raw import DaosContext, DaosPool, DaosContainer, DaosApiError, IOR
```

Comment:
Is `IOV` -> `IOR` an intended name change?

Reply:
Oops, good catch. thanks. fixed.